### PR TITLE
Subscribe Buttons in Header [#13]

### PR DESCRIPTION
**Pull request by LenaSchnedlitz**
This PR adds subscribe buttons to the header and implements #13.

The implementation is based on mockups that can be found in [in this **Figma file**](https://www.figma.com/file/RkxnaLyEmCQvd0eXCNmO9F/Header).

## Relevant Changes:

* Extracts all header related code into an own component in order to improve encapsulation and reduce side effects caused by global CSS.
* Adds a helper component to inline and style SVG icons (`SimpleIcon`)
* Adds the new subscribe section as discussed in #13.  
  **Please note:** Variant 2 of the mobile header has not been implemented. Please let me know if you would like to have that as well and I'll add open a follow-up PR 😉
* Fixes extend/collapse animation of mobile header.

### CODE REVIEW

File: src/styles/global.css [hunk 1]
```diff
@@ -71,202 +71,6 @@ p {
   margin: 0 auto;
 }
 
-.headers .gatsby-image-wrapper img {
```
Comment (LenaSchnedlitz) [inline src/styles/global.css:74]: This was moved to `header.module.css`

**Comment by LenaSchnedlitz** [comment]
I'm pretty sure the failed check is not related to my changes, but please let me know if I'm wrong.

**Comment by mgreiler** [comment]
This looks awesome. Thank you Lena.

**Comment by mgreiler** [comment]
> I'm pretty sure the failed check is not related to my changes, but please let me know if I'm wrong.

No, this failure is due to HTTP:// requests.

**Pull request merged**